analyzers/its/expected/Ember-MM/S2259-scraper.EmberCore.json

```diff
@@ -180,24 +180,12 @@
       "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/scraper.EmberCore/TVScraper/dlgTVDBSearchResults.vb#L336",
       "Location": "Line 336 Position 24-32"
     },
-    {
-      "Id": "S2259",
-      "Message": "\u0027sResults\u0027 is Nothing on at least one execution path.",
-      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/scraper.EmberCore/TVScraper/dlgTVDBSearchResults.vb#L347",
```

Comment:
[Link](https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/Ember-MM/Addons/scraper.EmberCore/TVScraper/dlgTVDBSearchResults.vb#L347)
`NotNull` should be learnt in `sResult.Select` in the `If` condition.